docs/contributing/running-tests.md

```diff
@@ -55,6 +55,12 @@ If you are on Windows and you want to run the tests using linux containers:
 USE_LINUX_CONTAINERS=1 ./gradlew :smoke-tests:test -PsmokeTestSuite=payara
 ```
 
+If you want to run a specific test from a specific suite:
```

Comment:
I think gradle terminology does not refer to this as `suite` https://blog.gradle.org/introducing-test-suites